### ISSUE TITLE
[wasm] WasmTemplateTests for console failing due to truncated output

### ISSUE THREAD

**Issue by radical** [dotnet/runtime#71887]
[Build](https://dev.azure.com/dnceng/public/_build/results?buildId=1869929&view=logs&jobId=55f4943c-f76d-585b-7250-deab324f0a54&j=f386e426-bcac-5c89-397e-589b8da0ec42&t=e6a4592a-90b9-5f43-2cfd-2dff490c67e2), and [log](https://dev.azure.com/dnceng/public/_build/results?buildId=1869929&view=logs&jobId=55f4943c-f76d-585b-7250-deab324f0a54&j=f386e426-bcac-5c89-397e-589b8da0ec42&t=e6a4592a-90b9-5f43-2cfd-2dff490c67e2):

```
   Wasm.Build.Tests: [Long Running Test] 'Wasm.Build.Tests.WasmTemplateTests.ConsolePublishAndRun', Elapsed: 00:02:08
    Wasm.Build.Tests.WasmTemplateTests.ConsolePublishAndRun(config: "Release", aot: True) [FAIL]
      Assert.Contains() Failure
      Not found: AOT: image 'Release_1rdp2lb0.wjl' found
      In value:  [Release_1rdp2lb0.wjl] WasmAppHost --runtime-config C:\helix\work\workitem\e\Release_1rdp2lb0.wjl\bin\Release\net7.0\browser-wasm\AppBundle\/Release_1rdp2lb0.wjl.runtimeconfig.json --setenv=MONO_LOG_MASK=aot --setenv=MONO_LOG_LEVEL=debug x y z
                 [Release_1rdp2lb0.wjl] [11:43:25] info: host[0] Running: C:\helix\work\correlation\build\emsdk-node/14.18.2_64bit/bin\node.exe main.mjs --setenv=MONO_LOG_MASK=aot --setenv=MONO_LOG_LEVEL=debug x y z
                 [Release_1rdp2lb0.wjl] [11:43:25] dbug: host[0] Using working directory: C:\helix\work\workitem\e\Release_1rdp2lb0.wjl\bin\Release\net7.0\browser-wasm\AppBundle
                 [Release_1rdp2lb0.wjl] [11:43:25] info: host[0] runArgs: {"applicationArguments":[],"runtimeArguments":[]}
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] Found statically linked AOT module 'System.Private.CoreLib'.
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: module System.Private.CoreLib wants to load image 0: System.Private.CoreLib
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: image 'System.Private.CoreLib' found.
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method (wrapper managed-to-native) object:__icall_wrapper_mono_threads_state_poll () [0x2a48 - 0x2a49 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method (wrapper managed-to-native) object:__icall_wrapper_mini_llvmonly_init_vtable_slot (intptr,int) [0x69 - 0x6a 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method (wrapper managed-to-native) object:__icall_wrapper_mono_helper_ldstr_mscorlib (int) [0x21a - 0x21b 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method (wrapper managed-to-native) object:__icall_wrapper_mono_fill_method_rgctx (intptr,int) [0x29f8 - 0x29f9 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method (wrapper managed-to-native) object:__icall_wrapper_mono_fill_class_rgctx (intptr,int) [0x1c7 - 0x1c8 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.OutOfMemoryException:.ctor (string) [0xb79 - 0xb7a 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.NullReferenceException:.ctor (string) [0xa8c - 0xa8d 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.StackOverflowException:.ctor (string) [0xe6c - 0xe6d 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.AppContext:Setup (char**,char**,int) [0x5e7 - 0x5e8 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method (wrapper other) object:gsharedvt_out_sig (object&,object&,int&,intptr) [0x2b1a - 0x2b1b 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.Collections.Generic.EqualityComparer`1:CreateComparer () [0x20fe - 0x20ff 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.Collections.HashHelpers:.cctor () [0x207a - 0x207b 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method (wrapper other) object:gsharedvt_out_sig (intptr) [0x2b1c - 0x2b1d 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.Collections.Generic.NonRandomizedStringEqualityComparer:.cctor () [0x219a - 0x219b 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.OrdinalIgnoreCaseComparer:.cctor () [0xe83 - 0xe84 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.OrdinalCaseSensitiveComparer:.cctor () [0xe7d - 0xe7e 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.Type:.cctor () [0x5b3 - 0x5b4 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.Type/<>c:.cctor () [0x5b4 - 0x5b5 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.Reflection.Missing:.cctor () [0x1bcb - 0x1bcc 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.Array/EmptyArray`1:.cctor () [0x2e8 - 0x2e9 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.Type:FilterAttributeImpl (System.Reflection.MemberInfo,object) [0x5b1 - 0x5b2 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method (wrapper delegate-invoke) <Module>:invoke_bool_MemberInfo_object (System.Reflection.MemberInfo,object) [0x2a71 - 0x2a72 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.Type/<>c:<.cctor>b__287_0 (System.Reflection.MemberInfo,object) [0x5b6 - 0x5b7 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.Type/<>c:<.cctor>b__287_1 (System.Reflection.MemberInfo,object) [0x5b7 - 0x5b8 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.RuntimeType:IsAssignableFrom (System.Type) [0x481 - 0x482 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.RuntimeType:.cctor () [0x485 - 0x486 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.RuntimeType:get_UnderlyingSystemType () [0x468 - 0x469 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method (wrapper stelemref) object:virt_stelemref_class_small_idepth (intptr,object) [0x29bc - 0x29bd 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.Reflection.ConstructorInfo:.cctor () [0x1b80 - 0x1b81 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method (wrapper stelemref) object:virt_stelemref_sealed_class (intptr,object) [0x29ba - 0x29bb 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.Reflection.MethodBase:GetParametersNoCopy () [0x1a41 - 0x1a42 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.Reflection.RuntimeConstructorInfo:GetParametersInternal () [0x1afd - 0x1afe 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.Array:Resize<T> (T[]&,int) [0x2d6 - 0x2d7 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.Reflection.RuntimeConstructorInfo:get_Attributes () [0x1b01 - 0x1b02 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.Collections.Generic.GenericEqualityComparer`1:.ctor () [0x2149 - 0x214a 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.Collections.Generic.Dictionary`2:Add (TKey,TValue) [0x2115 - 0x2116 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.ThrowHelper:ThrowAddingDuplicateWithKeyArgumentException<T> (T) [0xe9f - 0xea0 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: FOUND method System.Collections.Generic.NonRandomizedStringEqualityComparer/OrdinalComparer:GetHashCode (string) [0x219d - 0x219e 0x55068]
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] Found statically linked AOT module 'System.Runtime.InteropServices.JavaScript'.
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: module System.Runtime.InteropServices.JavaScript wants to load image 0: System.Runtime.InteropServices.JavaScript
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: module System.Runtime.InteropServices.JavaScript wants to load image 1: System.Private.CoreLib
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] AOT: module System.Runtime.InteropServices.JavaScript wants to load image 2: System.Private.Uri
                 [Release_1rdp2lb0.wjl] [11:43:30] info: host[0] [MONO] Found statically linked AOT module 'System.Private.Uri'.
      Stack Trace:
        /_/src/tests/BuildWasmApps/Wasm.Build.Tests/WasmTemplateTests.cs(210,0): at Wasm.Build.Tests.WasmTemplateTests.ConsolePublishAndRun(String config, Boolean aot)
           at InvokeStub_WasmTemplateTests.ConsolePublishAndRun(Object, Object, IntPtr*)
           at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
```

**Comment by lewing** [comment]
@radical do we know the cause of this?

**Comment by radical** [comment]
Yes. It's on my todo list, and I will get to it soon.

**Comment by radical** [comment]
This seems to be still failing.

**Comment by radical** [comment]
It's a testing issue which I will fix soon, but isn't *required* for 7.0 .

**Comment by lewing** [comment]
I think this is fixed now?
